catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/services/database/DatabaseServiceResource.java

```diff
@@ -338,9 +347,54 @@ public Response delete(
       throws IOException, ParseException {
     SecurityUtil.checkAdminOrBotRole(authorizer, securityContext);
     DeleteResponse<DatabaseService> response = dao.delete(securityContext.getUserPrincipal().getName(), id, recursive);
+    decrypt(securityContext, response.getEntity());
     return response.toResponse();
   }
 
+  @POST
+  @Path("/rotate")
```

Comment:
who will call this and what is necessity behind it.
My approach here is 
1. have a configurable key if needed in server config
2. If rotation is needed should be done through re-configuring the key or changing the file underneath it such that server restart is not required

Reply:
Done. Now we are using the CLI to rotate the key.